doc/readme/INTERFACES.md

```diff
@@ -69,6 +69,7 @@ This needs `shogun.R` to be visible, which is either in `path/to/build/src/inter
     export R_LIBS_USER="path/to/shogun.R:$R_LIBS_USER"
 
 Running an example:
+
     R --no-restore --no-save --no-readline --slave -f path/to/r_example.rb
```

Comment:
there's a typo here as well, i.e. it should be `path/to/r_example.r` and not `path/to/r_example.rb` (there's an extra `b` on the end)